src/Forms/GetDictionaries.cs

```diff
@@ -269,8 +269,7 @@ private static void ExtractDic(string dictionaryFolder, ZipExtractor zip, List<Z
 
         private void comboBoxDictionaries_SelectedIndexChanged(object sender, EventArgs e)
         {
-            int index = comboBoxDictionaries.SelectedIndex;
-            labelPleaseWait.Text = _descriptions[index];
+            labelPleaseWait.Text = ((OnlineDictionary)comboBoxDictionaries.SelectedItem).Description;

```

Comment:
it's better if we display the dictionary description instead of the url, which seem to be more interesting to the user.